src/modules/store/modules/layers.store.js

```diff
@@ -221,19 +221,20 @@ const actions = {
             }
         }
     },
-    showLayerPreview({ commit, getters }, layerId) {
-        const layer = getters.getLayerConfigById(layerId)
-        if (layer) {
-            const cloned = layer.clone()
-            cloned.visible = true
-            commit('setPreviewLayer', cloned)
+    setPreviewLayer({ commit, getters }, layerId) {
+        if (!layerId) {
+            commit('setPreviewLayer', null)
```

Comment:
Here if we implement a double event approach you could add a new action called `clearPreviewLayer` that would then call the mutation `setPreviewLayer` with `null` as param.

Reply:
Will do :+1: